src/components/ChatMessage.tsx

```diff
@@ -0,0 +1,35 @@
+import {
+    IonCard,
+    IonCardContent,
+    IonCardHeader,
+    IonImg,
+    IonCardTitle,
+} from "@ionic/react";
+import "./ChatMessage.scss";
+
+interface ChatMessageProps {
+    sent?: boolean,
+    src?: string,
+    text: string,
+    username?: string
```

Comment:
I don't think we ever want username to be null, so we should take out this ?.

Reply:
I assumed the username would be blank when the message is sent rather than received, but the options then are (a) always show username, regardless if message is sent or received or (b) take the username as an argument on sent messages but hide it in the HTML.

Reply:
I've checked a few different messaging apps and it looks like the standard is that the username is always displayed, regardless if message is sent or received.

Reply:
Done!